content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+
+What we have recently added is the `signature` part:
+
+```yaml
+signature:
+  cosign:
+    certificate-oidc-issuer: https://token.actions.githubusercontent.com
+    certificate-identity-regexp: https://github.com/falcosecurity/plugins/
+```
+
+This section indicates how [to check the signature](https://docs.sigstore.dev/signing/overview/#verifying-the-signed-artifact) for this artifact. This way the index file contains information on how to download and how to verify the artifact.
+
+Upon request to install the artifact `falcoctl` is going to check the index file, resolve the name, identify the digest from the registry, pull the signature, and validate it by using cosign as a library. Only once the signature is verified, falcoctl will allow the installation of the plugin with that specific digest. If the signature data does not match the installation will be halted with an error.
+
+You can try it yourself to make sure that the signature works!
+
+```
+sudo docker run -it --entrypoint sh falcosecurity/falcoctl:0.6.1
+# falcoctl artifact install cloudtrail:0.9 --rulesfiles-dir=/tmp --plugins-dir=/tmp
+```
+
+![](https://hackmd.io/_uploads/Byldjgnla.png)
+
+But there is more! If you have `cosign` installed you can independently verify the signature without using falcoctl or any index file:
+
+```
+$ cosign verify ghcr.io/falcosecurity/plugins/plugin/cloudtrail:0.9.0 --certificate-oidc-issuer=https://token.actions.githubusercontent.com --certificate-identity-regexp=https://github.com/falcosecurity/plugins/
+
+Verification for ghcr.io/falcosecurity/plugins/plugin/cloudtrail:0.9.0 --
+The following checks were performed on each of these signatures:
+  - The cosign claims were validated
+  - Existence of the claims in the transparency log was verified offline
+  - The code-signing certificate was verified using trusted certificate authority certificates
+
+[...]
+```
+
+This is one of the main advantages of relying on already existing and popular container artifacts signing technologies of the cloud-native world. Authentication of the artifacts is decoupled from the producer, and consumers can always choose the method they prefer.
```

Comment:
Try this for the first sentence. 

This is one of the main advantages of relying on widely adopted cloud native contain artifact signing technologies.

Reply:
I like it @mikegcoleman, thank you a lot for this review!

Reply:
```suggestion
This is one of the main advantages of relying on widely adopted container artifacts signing technologies of the cloud-native world. Authentication of the artifacts is decoupled from the producer, and consumers can always choose the method they prefer.
```